test/greet.test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const assert = require('assert');
+const greet = require('../lib/greet');
+
+const name = 'steven';
+
+describe('greet.js', function(){
+  describe('#greet', function(){
+    it('is a string', function(){
+      assert.equal(typeof greet.greet(name), 'string');
+    });
+    it('recieves the argument', function(){
+      assert.strictEqual(greet.greet(name), 'Hello steven');
```

Comment:
From what I gather, deepEqual tests own propeties as well as child properties, and evaluates with a regular == operator. With strictEqual, it tests only that object's own properties and does a strict evaluation with === which will also compare types.